symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/util/PresentationMLParser.java

```diff
@@ -59,6 +59,7 @@ public static String getTextContent(String presentationML) throws PresentationML
   @SneakyThrows
   private static DocumentBuilder initBuilder() {
     final DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+    factory.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
```

Comment:
why this change?

Reply:
It s a security issue fix, proposed by the the security scanning action.